client/public/manifest.json

```diff
@@ -1,21 +1,11 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "Virtual Big Techday 13",
```

Comment:
I guess we don't want this here.

- [ ] Remove me

Reply:
```suggestion
  "short_name": "Virtual Office",
```
Like this?